SendGrid/src/SendGrid.Sample/Example.cs

```diff
@@ -22,7 +22,7 @@ private static void Main()
 
         private static async Task HelloEmail()
         {
-            String apiKey = Environment.GetEnvironmentVariable("SENDGRID_APIKEY", EnvironmentVariableTarget.User);
+            String apiKey = Environment.GetEnvironmentVariable("SENDGRID_APIKEY");
```

Comment:
'cause that sample add is dotnet portable app, not have specific runtime, so you cound run with Environment Variable by using "SET" and "dotnet run" command. In last commit (https://github.com/sendgrid/sendgrid-csharp/pull/320/commits/3282e5cae21c7494886b1d7c45fb4c62b29e8002), I have change sample app to Standard .net Console app in .net 4.5.2.